addon/components/attach-popover.js

```diff
@@ -301,9 +300,9 @@ export default class AttachPopover extends Component {
 
   @action
   _ensureArgumentsAreValid() {
-    stripInProduction(() => {
+    runInDebug(() => {
```

Comment:
It seems that [runInDebug](https://api.emberjs.com/ember/release/functions/@ember%2Fdebug/runInDebug) is removed from production build